debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -1368,17 +1381,36 @@ private static int validateFlushLsnSource(Configuration config, Field field, Fie
         }
         return 0;
     }
+    
+    public static Pair<String, String> findAndReplaceLoadBalancePropertyValues(Boolean load_balance) {
+        String multiHostUrl = PostgresConnection.MULTI_HOST_URL_PATTERN;
+        String singleHostUrl = PostgresConnection.URL_PATTERN;
+        String value = load_balance.toString();
+
+        if (multiHostUrl.contains("${" + PostgresConnectorConfig.YB_LOAD_BALANCE_CONNECTIONS + "}")) {
+            multiHostUrl = multiHostUrl.replaceAll(
+                    "\\$\\{" + PostgresConnectorConfig.YB_LOAD_BALANCE_CONNECTIONS + "\\}", value);
+        }
+
+        if (singleHostUrl.contains("${" + PostgresConnectorConfig.YB_LOAD_BALANCE_CONNECTIONS + "}")) {
```

Comment:
I do not understand the if conditions for both multi and single host URL - why would they contain the name `yb.load.balance.connections`?

Reply:
I have added the `${yb.load.balance.connections}` in both multi and single host URL patterns. 

When this property is set to true the driver fetches and maintains a list of nodes from the given endpoint and distributes the connections equally across these nodes. Earlier we were using this only in case of multi host url pattern. Now we can use it for both the patterns.